src/pages/index.tsx

```diff
@@ -201,7 +202,11 @@ export const getStaticProps: GetStaticProps = async ({ params }) => {
 
   const recentTagsByUsers = await getMediaForFeed(20, 3)
 
-  const recentTags = recentTagsByUsers.flatMap(entry => entry.mediaWithTags)
+  const areaTags = recentTagsByUsers.flatMap(entry => entry.mediaWithTags)
+  const recentTags = areaTags.filter(tag => {
+    return !tag.entityTags.some(entityTag =>
+      testAreaIds.has(entityTag.targetId))
+  })
```

Comment:
This filter will correctly exclude media with the exact match of NEXT_PUBLIC_TEST_AREA_IDS. 
There's another scenario where we have media tagged in child areas or climbs under 'NEXT_PUBLIC_TEST_AREA_IDS'.   You can use regular expression to match `testAreaIds` and `entityTag.ancestors`.

Example:
A test area (and its children) we want to exclude (NZ test area)
- https://stg.openbeta.io/crag/f72326ab-154b-527b-8bf6-4f9b73e7467c

A photo tagged under `NZ test area / NZ Sport & Trad`
- https://stg.openbeta.io/climbs/2d63fb47-b2eb-42f1-af9f-26d4b7a1ee5a

Reply:
You'll have to forgive my ignorance as I am quite new to programming, but could you provide an example of this regex?

I do not completely understand what you are asking.